src/pepr/operator/controllers/keycloak/client-sync.ts

```diff
@@ -66,14 +70,26 @@ export async function purgeSSOClients(pkg: UDSPackage, refs: string[] = []) {
     if (token) {
       Store.removeItem(ref);
       await apiCall({ clientId }, "DELETE", token);
+
+      // find sso by clientId
+      const sso = pkg.spec?.sso?.find(sso => sso.clientId === clientId);
+
+      // if sso.enableAuthserviceSelector was set, remove from authservice config
+      if (sso && sso.enableAuthserviceSelector) {
```

Comment:
This conditional currently doesn't work due to checking the "active" spec for the client ID vs the "previous state". We'll need to make sure we have a way to check the previous state to determine if something was an authservice client and got removed/modified.